ntbs-service-unit-tests/Services/NotificationServiceTest.cs

```diff
@@ -430,6 +431,36 @@ private void VerifyUpdateDatabaseCalled()
                 mock.SaveChangesAsync(It.IsAny<NotificationAuditType>(), It.IsAny<string>()));
         }
 
+        [Fact]
+        public async Task CreateNewNotificationForUserAsync_CreatesCaseUsingUserDetails()
+        {
+            // Arrange
+            const string serviceCode = "TBS0008";
+            const string username = "User1";
+            var userId = 1234;
+            var tbService = new TBService
+            {
+                ServiceAdGroup = "ServiceAshford",
+                Name = username,

```

Comment:
I don't think the `ServiceAdGroup` and `Name` are really needed for this test, so I would probably ignore them - especially the `Name` given that it's currently set to the user name!

Reply:
Yep, thanks for catching that! Both have been removed